sdk/storage/storage-file/test/aborter.spec.ts

```diff
@@ -46,34 +41,27 @@ describe("Aborter", () => {
   });
 
   it("Should not abort when calling abort() after request finishes", async () => {
-    const aborter = Aborter.none;
+    const aborter = new AbortController();
     await shareClient.create();
     aborter.abort();
   });
 
   it("Should abort after aborter timeout", async () => {
     try {
-      await shareClient.create({ abortSignal: Aborter.timeout(1) });
+      await shareClient.create({ abortSignal: AbortController.timeout(1) });
       assert.fail();
     } catch (err) {}
   });
 
   it("Should abort after parent aborter calls abort()", async () => {
     try {
-      const aborter = Aborter.none;
-      const response = shareClient.create({ abortSignal: aborter.withTimeout(10 * 60 * 1000) });
+      const aborter = new AbortController();
+      const response = shareClient.create({
+        abortSignal: AbortController.timeout(10 * 60 * 1000)
+      });
       aborter.abort();
       await response;
       assert.fail();
     } catch (err) {}
   });
-
-  it("Should abort after parent aborter timeout", async () => {
-    try {
-      const aborter = Aborter.timeout(1);
-      const response = shareClient.create({ abortSignal: aborter.withTimeout(10 * 60 * 1000) });
```

Comment:
Removed this test since withTimeout is removed from the new abort-controller package and I couldn't find any equivalent.